### PR TITLE
21 test kateryna

### PR DESCRIPTION
**Pull request by KaterynaKs**
pull request 21

### CODE REVIEW

**Review by ella-kadriaeva — COMMENTED** [review]
Alles gut!

**Review by ella-kadriaeva — APPROVED** [review]

File: SomeFile.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 // site for "NiceMedia" - video production company
 // Comment from Danylo
-// Change from Teodor
\ No newline at end of file
+// Change from Teodor
+// Local
```
Comment (Piterom911) [inline SomeFile.js:4]: Please use getElementById instead of nothing.

**Review by Piterom911 — CHANGES_REQUESTED** [review]
Fix the bug)))

File: SomeFile.js (hunk 1 above)
Comment (KaterynaKs) [reply SomeFile.js]: Corrected)

**Review by Piterom911 — APPROVED** [review]
Perfect!!!

**Pull request merged**